games-fps/chocolate-doom/chocolate_doom-3.0.0.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="Doom source port that accurately reproduces the experience of Doom"
+DESCRIPTION="Chocolate Doom is a source port of the game Doom, by id Software. \
+The purpose of Chocolate Doom is to be as compatible as possible with the \
+original DOS version of Doom."
+HOMEPAGE="https://www.chocolate-doom.org/"
+COPYRIGHT="Id Software Inc. 1993-1996
+	Simon Howard
+	James Haley
+	Samuel Villarreal"
```

Comment:
This should be updated to conform to the Packaging Policy. Thanks to folks at Debian, [here][0] is the list of copyright holders and their associated years.

```diff
-COPYRIGHT="Id Software Inc. 1993-1996
-	Simon Howard
-	James Haley
-	Samuel Villarreal"
+COPYRIGHT="1993-1996 Id Software, Inc
+	1993-2008 Raven Software
+	1996 Rogue Entertainment / Velocity, Inc.
+	1996-2013 Free Software Foundation, Inc
+	1999 Thomas A. Fine
+	2002-2003 Marcel Telka
+	2002-2010 The DOSBox Team
+	2005-2014 Simon Howard
+	2005-2014 Andrey Budko
+	2006 Ben Ryves
+	2008 GhostlyDeath
+	2008 David Flater
+	2010 James Haley
+	2010 Samuel Villarreal
+	2011 Team Eternity
+	2014 Fabian Greffrath
+	2001-2008 Contributors to the Freedoom project"
```

[0]: http://metadata.ftp-master.debian.org/changelogs/main/c/chocolate-doom/chocolate-doom_3.0.0-3_copyright